### PR TITLE
docs: correct authorization directive composition

### PR DESCRIPTION
**Pull request by Meschreiber**
Makes authorization directive composition docs clearer

### CODE REVIEW

File: docs/source/errors.mdx [hunk 1]
```diff
@@ -94,14 +94,13 @@ The actual cost of the query was greater than the configured maximum cost.
...
 
 </Property>
- <Property name="COST_RESPONSE_TYPING_FAILURE">
+<Property name="COST_RESPONSE_TYPING_FAILURE">
 
 The response from a subgraph did not match the GraphQL schema.
 
 </Property>
 
-</Property>
```
Comment (Meschreiber) [inline docs/source/errors.mdx:103]: This additional `</Property>` was causing builds to fail, so corrected it here.

File: docs/source/configuration/authorization.mdx [hunk 2]
```diff
@@ -688,12 +688,12 @@ When using subscriptions along with `@policy` authorization, subscription events
...
+GraphOS's composition strategy for authorization directives is intentionally accumulative. When you define authorization directives on fields and types in subgraphs, GraphOS composes them into the supergraph schema. In other words, if subgraph fields or types include `@requiresScopes`, `@authenticated`, or `@policy` directives, they are set on the supergraph too. Whether composition uses `AND` or `OR` logic depends on how the authorization directives are used.
 
-#### Composition with `AND`/`OR` logic
-
-If shared subgraph fields include multiple directives, composition merges them. For example, suppose the `me` query requires `@authentication` in one subgraph:
+### Composed fields with different authorization directives
 
+If a shared field uses different authorization directives across subgraphs, composition merges them using `AND` logic.
+For example, suppose the `me` query requires `@authentication` in one subgraph and the `read:user` scope in another subgraph:
```
Comment (goto-bus-stop) [inline docs/source/configuration/authorization.mdx:696]:
```suggestion
For example, suppose the `me` query requires `@authenticated` in one subgraph and the `read:user` scope in another subgraph:
```

**Review by goto-bus-stop — APPROVED** [review]
this matches my understanding!

**Comment by Meschreiber** [comment]
!docs preview

**Pull request merged**